src/classes/CW_CampaignRewardTriggerHandler.cls

```diff
@@ -0,0 +1,30 @@
+public with sharing class CW_CampaignRewardTriggerHandler 
+{
+	private Boolean m_isExecuting = false;
+
+    public CW_CampaignRewardTriggerHandler(Boolean isExecuting){
+        m_isExecuting = isExecuting;
+    }
+    
+    public void OnBeforeUpdate(Map<Id,CW_Campaign_Reward__c> oldMap, Map<Id,CW_Campaign_Reward__c> newMap)
```

Comment:
Style issue: methods should use the "lowerCamelCase" notation. Suggestion: rename accordingly.